recipe_scrapers/_utils.py

```diff
@@ -191,6 +191,7 @@ def normalize_string(string):
         unescaped_string.replace("\xc2\xa0", " ")
         .replace("\xa0", " ")
         .replace("\u200b", "")
+        .replace("\r", " ")
```

Comment:
Thanks @marian-code for this pull request!

Could we replace `\r\n` here instead?  There are two things I have in mind:

  * This could handle cr-lfs without affecting any (unexpected, but we should leave them as-is to disover them) singleton carriage-returns.
  * Using a two-character match string _might_ allow the replacement algorithm to use a two-character-step approach for most comparisons.  The code here runs a lot, and although we don't performance-test it (should do), I think it's worth being a little careful about.

Reply:
My idea was that the carriage returns are not needeed anyway so I might just remove them. But your point of view makes sense so let's do it that way :)

Reply:
There ai also a https://docs.python.org/3/library/unicodedata.html#unicodedata.normalize function which could be useful here, but that could mess with the current tests.

Reply:
About the performance part, that is not an issue. Sure string replacements are costly as string is imutable, but the strings that are common here are short. I actually took a look and the Beautifulsoups html parser is where the code spends roughly 90% of time (see picture below). So if the code should be optimized it should be there by switching to [lxml] :) but that will messup everything since the persers do not output exactly identical results :( (https://beautiful-soup-4.readthedocs.io/en/latest/index.html?highlight=lxml#improving-performance)
![image](https://github.com/hhursev/recipe-scrapers/assets/26881798/e610469b-5d12-4014-8c18-c77217c30dc7)